python/ola/ClientWrapper.py

```diff
@@ -47,6 +47,21 @@ def __init__(self, time_ms, callback):
   def __cmp__(self, other):
     return cmp(self._run_at, other._run_at)
 
+  def __eq__(self, other):
+    return self._run_at == other._run_at and self._callback == other._callback
+
+  def __lt__(self, other):
```

Comment:
Dunno how we should handle these bits then though...

Reply:
Again this is above and beyond or matching Python:
https://github.com/python/cpython/blob/0baa72f4b2e7185298d09cf64c7b591efcd22af0/Lib/asyncio/events.py#L97-L145